scripts/watchdog

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+VERBOSE=no
+ARISTA_WD_TIMER=180
+ACTION=none
+ENABLE=enable
+DISABLE=disable
+
+function debug()
+{
+    if [[ x"${VERBOSE}" == x"yes" ]]; then
+        echo "$date $@"
+    fi
+    logger "$@"
+}
+
+function parseMachineConf() {
+    ONIE_PLATFORM="$(cat /host/machine.conf | grep 'onie_platform=' | cut -f2 -d'=')"
+    ABOOT_PLATFORM="$(cat /host/machine.conf | grep 'aboot_platform=' | cut -f2 -d'=')"
+    ONIE_SWITCH_ASIC="$(cat /host/machine.conf | grep 'onie_switch_asic=' | cut -f2 -d'=')"
+}
+
+function isDellS6100() {
+    local -r _DELLEMC_S6100_PLATFORM="x86_64-dell_s6100"
+
+    if [[ $ONIE_PLATFORM =~ $_DELLEMC_S6100_PLATFORM ]]; then
+        return 0
+    fi
+
+    return 1
+}
+
+function isMLNX() {
+    local -r _MELLANOX_PLATFORM="mlnx"
+
+    if [[ "${ONIE_SWITCH_ASIC}" = "${_MELLANOX_PLATFORM}" ]]; then
+        return 0
+    fi
+
+    return 1
+}
+
+
+function isArista() {
+    local -r _ARISTA_PLATFORM="x86_64-arista"
+
+    if [[ "${ABOOT_PLATFORM}" =~ "${_ARISTA_PLATFORM}" ]]; then
```

Comment:
Why quote the operands here, but not in the regex compare in isDellS6100() above? I suggest making them match. Either add quotes in isDellS6100() or remove the quotes here.